actors/migration/nv15/top.go

```diff
@@ -226,10 +238,9 @@ func MigrateStateTree(ctx context.Context, store cbor.IpldStore, actorsRootIn ci
 
 type actorMigrationInput struct {
 	address    address.Address // actor's address
-	balance    abi.TokenAmount // actor's balance
-	head       cid.Cid         // actor's state head CID
-	priorEpoch abi.ChainEpoch  // epoch of last state transition prior to migration
-	cache      MigrationCache  // cache of existing cid -> cid migrations for this actor
+	head       cid.Cid
```

Comment:
balance shouldn't have been in here, i don't think

Reply:
I think we needed it for some migrations in the past.